modules/users/tests/server/user-block.server.routes.tests.js

```diff
@@ -0,0 +1,279 @@
+const should = require('should');
+const request = require('supertest');
+const path = require('path');
+const mongoose = require('mongoose');
+const User = mongoose.model('User');
+const express = require(path.resolve('./config/lib/express'));
+
+/**
+ * Globals
+ */
+let app;
+let agent;
+
+/* users of the test */
+let alice;
+let bob;
+let carol;
+
+/**
+ * User routes tests
+ */
+describe('User block - user', function () {
+  before(function (done) {
+    // Get application
+    app = express.init(mongoose.connection);
+    agent = request.agent(app);
+
+    done();
+  });
```

Comment:
I know this is how most of the app does this, but it can also be simplified nowadays to this:

https://github.com/Trustroots/trustroots/blob/aaa8b9ed0269c8e4ea157bcc6097db18b7eb5719/modules/admin/tests/server/admin.audit-log.server.routes.tests.js#L8-L10

Not a blocker for merging.

Reply:
Nice! Much more simple.